### PR TITLE
Replace SheetBase

### LINKED ISSUE #167: Replace Deprecated in DisplayLanguage.qml
plugins/language/DisplayLanguage.qml still uses `SheetBase` which is deprecated and looks old. Replace by ¿`Page`?

### PR DESCRIPTION
**Pull request by mateosalta**
fix https://github.com/ubports/system-settings/issues/167

- replace SheetBase 
- update items to work
- theme-able

Todo:
- remove bfb's in favor of header buttons

### CODE REVIEW

**Comment by mateosalta** [comment]
er, work in progress... don't merge yet

**Comment by UniversalSuperBox** [comment]
Please add WIP to the title and ping me when it's no longer WIP. :)

**Comment by mateosalta** [comment]
@UniversalSuperBox 
Ok, I think this is in a workable state. hopefully I haven't made typos.
This will get us away from the very old looking page, into something that basically works the same but looks new :)

screenshot20191119_213803133
screenshot20191119_213742204
screenshot20191119_212312904

**Comment by UniversalSuperBox** [comment]
I've found a rather strange issue... All I have to do is open the language page and it appears.

image

**Comment by mateosalta** [comment]
ah, I think I missed something. I will compare

File: plugins/language/DisplayLanguage.qml [hunk 1]
```diff
@@ -24,30 +24,41 @@ import Ubuntu.Components.Popups 1.3
...
 
+  
+
     ListView {
         id: languageList
         objectName: "languagesList"
         clip: true
 
-        anchors.top: parent.top
+        anchors.top: head.bottom
```
Comment (mateosalta) [inline plugins/language/DisplayLanguage.qml:61]: this should fix the overlapping header

**Comment by mateosalta** [comment]
@UniversalSuperBox Got it, I missed a anchor, where I needed to anchor the list below the header :)

**Review by UniversalSuperBox — APPROVED** [review]
Very nice! Good work.

**Pull request merged**